### DISCUSSION TITLE
网页端实现移动端的“我不想看”

### DISCUSSION THREAD

**Discussion by H-XZ** [the1812/Bilibili-Evolved#4224] [功能建议 / Ideas]
网页端推送机制过于稀烂，还没有不感兴趣，踩之类的负向反馈，有可能通过脚本实现和移动端类似的功能吗

**Comment by Geeyun-JY3** [comment]
如果是指首页推荐的「我不想看」：https://github.com/the1812/Bilibili-Evolved/discussions/2913#discussioncomment-2056033

**Reply by H-XZ** [reply]
感谢，非常有用，完美解决问题

**Comment by Geeyun-JY3** [comment]
关于进入视频播放页后的「不感兴趣」「踩」：https://github.com/the1812/Bilibili-Evolved/discussions/3664#discussioncomment-3636398
